spec/lib/services/nodes-api-service-spec.js

```diff
@@ -499,13 +506,24 @@ describe("Http.Services.Api.Nodes", function () {
             .to.equal(undefined);
         });
 
-        it("_removeRelation should fail if don't have targets", function() {
+        it("_removeRelation should return undefined if don't have relations", function() {
+            var enclNode = {
+                    id: '1234abcd1234abcd1234abcd',
+                    type: 'enclosure'
+                };
+
+            expect(nodeApiService._removeRelation(enclNode, 'encloses', computeNode))
```

Comment:
@yyscamper There's still debates for unit test for private methods. my opinion is no objection for this as long as the codes are qualified :)